src/schemas/user.schema.js

```diff
@@ -2,15 +2,57 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var userSchema = new Schema({
-  name: {
+  Id: {
     type: String,
     required: true
   },
+  Fname: {
+    type: String,
+    required: true
+  },
+  Mname: {
+    type: String,
+    required: false
+  },
+  Lname: {
+    type: String,
+    required: true
+  },
+  Bday: {
+    type: String,
+    required: true
+  },
+  Gender: {
+    type: String,
+    required: true
+  },
+  CivilStatus: {
+    type: String,
+    required: true
+  },
+  Bplace: {
+    type: String,
+    required: true
+  },
+  MobCellNumber: {
+    type: String,
+    required: false
+  },
+
   email: {
     type: String,
     required: true,
     unique: true
   },
+  Address: {
+    type: String,
```

Comment:
@melvinpurio this should be another schema like this.

`var addressSchema = new mongoose.Schema({

	street	: {type: String},
	city 	: {type: String},
	state 	: {type: String},
	country : {type: String},
	zip 	: {type: Number}
} ,{ _id : false });`

then the type will be `{ type: addressSchema }`

Reply:
Can explain why we should exclude the address from user schema?

Reply:
because we should probably reuse address schema for example an address for a certain place.
one scenario here is the bus terminal address

Reply:
by isolating it we can export it on other schemas

Reply:
Oh yes you are right

Reply:
hmm we should create address.schema.ts file and export it. then import it here in user schema

Reply:
Ok got it